module/Api/src/Domain/Repository/Application.php

```diff
@@ -53,7 +54,8 @@ private function prepareFetchByOrgAndStatus($orgId, array $statuses = [])
         $qb = $this->createQueryBuilder();
 
         $this->getQueryBuilder()->modifyQuery($qb)
-            ->withRefdata();
+            ->withRefdata()
+            ->with('licence');
```

Comment:
This method is quite old and has never had the ->with('licence'); in it before, so this will now be including licence data for anything else dependent on this repo method. Are we sure this wont have any detrimental effects elsewhere? Possibly safer adding some conditional behaviour that selectively adds the with()? Im not sure just flagging for discussion.

Reply:
Swappped for more appropriate one and reverted this line